src/components/testnet-badge/testnet-badge.tsx

```diff
@@ -37,7 +37,8 @@ export const TestnetBadgeComponent = (
                 <Text style={props.styles.text}>
                     {translate('App.labels.youAreOn', {
                         blockchain: props.blockchain ? Capitalize(props.blockchain) : '',
-                        networkName: props.networkName
+                        networkName: props.networkName === 'Testnet' ? '' : props.networkName,
+                        space: props.networkName === 'Testnet' ? '' : ' '
```

Comment:
Let's don't add this logic.

I think it's easier to just make a verification of the final translate and remove the double occurrences of words (such as Testnet).

Reply:
Okay sure